analytics/settings.toml

```diff
@@ -0,0 +1,4 @@
+NAME = "app"
+HOST = "0.0.0.0"
+USER = "app"
+PORT = 5432
```

Comment:
⭐⭐  &nbsp; _(optional but seriously consider)_ recommend prefixing all of these with `POSTGRES_`, eg. `POSTGRES_USER`